src/snapshot/SnapshotMatcher.lua

```diff
@@ -0,0 +1,101 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Markers = require(script.Parent.Serialize.Markers)
+local Serialize = require(script.Parent.Serialize)
+local deepEqual = require(script.Parent.Parent.deepEqual)
+local ElementKind = require(script.Parent.Parent.ElementKind)
+
+local SnapshotFolderName = "RoactSnapshots"
```

Comment:
minor: should be loud snake case SNAPSHOT_FOLDER_NAME for this local constant

Reply:
should I make it configurable? or do we push that to another PR?

Reply:
Doesn't need to be configurable right now